packages/design-tokens/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Migrate `@metamask/design-tokens` into the design system monorepo ([128](https://github.com/MetaMask/metamask-design-system/pull/128))
+
```

Comment:
Aligns with step 5 https://github.com/MetaMask/metamask-design-system/blob/main/docs/package-migration-process-guide.md#5-record-changes-made-to-any-metamask-design-system-package-in-its-changelog-under-the--unreleased-heading
<img width="1072" alt="Screenshot 2024-11-18 at 2 30 36 PM" src="https://github.com/user-attachments/assets/5cce1c43-f1ec-424c-9259-133a905bf875">